<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseApplicationForm</class>
 <widget class="QMainWindow" name="BaseApplicationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="te::qt::widgets::MultiThreadMapDisplay" name="m_display" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="m_exitAction"/>
   </widget>
   <addaction name="m_menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="m_fileToolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_addLayerAction"/>
  </widget>
  <widget class="te::qt::widgets::LayerExplorer" name="m_layerExplorer">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
  </widget>
  <widget class="QToolBar" name="drawBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_drawAction"/>
   <addaction name="m_fitLayersAction"/>
  </widget>
  <widget class="QToolBar" name="basicTools_bar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_zoomInAction"/>
   <addaction name="m_panAction"/>
   <addaction name="m_selectToolAction"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="m_addLayerAction">
   <property name="icon">
    <iconset theme="datasource">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Add Layer...</string>
   </property>
  </action>
  <action name="m_drawAction">
   <property name="icon">
    <iconset theme="map-draw">
     <normaloff>../../../../../share/terralib/icons/terralib/scalable/map-draw.svg</normaloff>../../../../../share/terralib/icons/terralib/scalable/map-draw.svg</iconset>
   </property>
   <property name="text">
    <string>Draw...</string>
   </property>
   <property name="toolTip">
    <string>Draw visible layers...</string>
   </property>
  </action>
  <action name="m_zoomInAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="zoom-area">
     <normaloff>../../../../../share/terralib/icons/terralib/scalable/zoom-area.svg</normaloff>../../../../../share/terralib/icons/terralib/scalable/zoom-area.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom in..</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="m_panAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="pan">
     <normaloff>../../../../../share/terralib/icons/terralib/scalable/pan.svg</normaloff>../../../../../share/terralib/icons/terralib/scalable/pan.svg</iconset>
   </property>
   <property name="text">
    <string>Pan...</string>
   </property>
   <property name="toolTip">
    <string>Pan</string>
   </property>
  </action>
  <action name="m_selectToolAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="pointer-selection">
     <normaloff>../../../../../share/terralib/icons/terralib/scalable/pointer-selection.svg</normaloff>../../../../../share/terralib/icons/terralib/scalable/pointer-selection.svg</iconset>
   </property>
   <property name="text">
    <string>Select features...</string>
   </property>
   <property name="toolTip">
    <string>Select features...</string>
   </property>
  </action>
  <action name="m_fitLayersAction">
   <property name="icon">
    <iconset theme="zoom-extent">
     <normaloff>../../../../../share/terralib/icons/terralib/scalable/zoom-extent.svg</normaloff>../../../../../share/terralib/icons/terralib/scalable/zoom-extent.svg</iconset>
   </property>
   <property name="text">
    <string>Fit layers...</string>
   </property>
   <property name="toolTip">
    <string>Fit...</string>
   </property>
  </action>
  <action name="m_exitAction">
   <property name="icon">
    <iconset theme="system-log-out">
     <normaloff>../../../../../share/terralib/icons/terralib/scalable/system-log-out.svg</normaloff>../../../../../share/terralib/icons/terralib/scalable/system-log-out.svg</iconset>
   </property>
   <property name="text">
    <string>Exit...</string>
   </property>
   <property name="toolTip">
    <string>Quit the application</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>te::qt::widgets::LayerExplorer</class>
   <extends>QDockWidget</extends>
   <header>terralib/qt/widgets/layer/explorer/LayerExplorer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>te::qt::widgets::MultiThreadMapDisplay</class>
   <extends>QWidget</extends>
   <header>terralib/qt/widgets/canvas/MultiThreadMapDisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_addLayerAction</sender>
   <signal>triggered()</signal>
   <receiver>BaseApplicationForm</receiver>
   <slot>onAddLayerTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_drawAction</sender>
   <signal>triggered()</signal>
   <receiver>BaseApplicationForm</receiver>
   <slot>onDrawTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_zoomInAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>BaseApplicationForm</receiver>
   <slot>onZoomInTriggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_panAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>BaseApplicationForm</receiver>
   <slot>onPanTriggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_selectToolAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>BaseApplicationForm</receiver>
   <slot>onSelectionTriggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fitLayersAction</sender>
   <signal>triggered()</signal>
   <receiver>BaseApplicationForm</receiver>
   <slot>onFitLayersTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_exitAction</sender>
   <signal>triggered()</signal>
   <receiver>BaseApplicationForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddLayerTriggered()</slot>
  <slot>onDrawTriggered()</slot>
  <slot>onZoomInTriggered(bool)</slot>
  <slot>onPanTriggered(bool)</slot>
  <slot>onSelectionTriggered(bool)</slot>
  <slot>onFitLayersTriggered()</slot>
 </slots>
</ui>
